netperf_tool_interfaces/msg/Bytes.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time  timestamp
+uint64  id
+byte[]  data
```

Comment:
Nitpick: Why are there two spaces between items here? Seems weird to me. Code linters from other languages would complain I think.

Reply:
I don't know why I was using two spaces :joy:.
Fixed in https://github.com/ros2/ros2cli/pull/720/commits/923e5af693109d3772e234b85dd5fa88a4b15f72